docs/extra-build-config.md

```diff
@@ -278,6 +278,14 @@ the header extension should set the following in local.conf:
 
     ENABLE_DWC2_HOST = "1"
 
+## Enable SSH support
+
+By default the raspberry pi's disable ssh. By setting the following variable
+we create the empty file "ssh" in the /boot/ directory which is used to enable
```

Comment:
What "consumes" this `ssh` file?

Reply:
Yep, good question. It turns out (as you likely know) nothing.

Reply:
RIght. So it must be some mechanism from another distro. Was it Raspbian?

Reply:
Yeah, that is where I got the idea from. Following their lead I added it to my Yocto images. I did not actually double check that it was necessary.